connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -0,0 +1,588 @@
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "group": "authentication",
+      "type": "Hidden",
+      "value": "noAuth",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.authentication.type"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "id": "tokenAuthType",
+      "group": "authentication",
+      "type": "Hidden",
+      "value": "noAuth",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.type"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "tokenAuth"
+        ]
+      }
+    },
+    {
+      "description": "method for authentication",
+      "type": "Hidden",
+      "value": "post",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.method"
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Username",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.username"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "oneOf": [
+          "userNamePasswordAuth",
+          "userNameApiKeyAuth"
+        ]
+      }
+    },
+    {
+      "label": "Password",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.password"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNamePasswordAuth"
+      }
+    },
+    {
+      "label": "API key",
+      "group": "authentication",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "authentication.request.body.apiKey"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "authenticationType",
+        "equals": "userNameApiKeyAuth"
+      }
+    },
+    {
+      "label": "Multiple login",
+      "group": "authentication",
+      "description": "Allow or disallow multiple login. For more information on multi-login, see <a href=\"https://docs.camunda.io/docs/components/modeler/web-modeler/connectors/available-connectors/automation-anywhere/\" target=\"_blank\">documentation</a>",
```

Comment:
this description is not helpfull. It doesn't describe what multiple login means.
If we can't describe it in one sentence, then just reference the documentation like:
for more information see <link>